vital/algo/detected_object_set_input.cxx

```diff
@@ -88,13 +90,13 @@ ::open( std::string const& filename )
   }
 
   // try to open the file
-  std::istream* file( new std::ifstream( filename ) );
-  if ( ! file )
+  std::unique_ptr< std::istream > file( new std::ifstream( filename ) );
+  if ( ! *file )
   {
     throw kwiver::vital::file_not_found_exception( filename, "open failed"  );
```

Comment:
Just a minor comment - This could be `VITAL_THROW` .

Reply:
These should all be converted to use `VITAL_THROW` now.